<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScalesContainer</class>
 <widget class="QWidget" name="ScalesContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="scaleName">
       <property name="placeholderText">
        <string>Введите название новой шкалы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addScalesBtn">
       <property name="toolTip">
        <string>Шкала будет добавлена как в ключ,
так и в оценочную карту</string>
       </property>
       <property name="text">
        <string>Добавить шкалу</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="keyTab">
      <attribute name="title">
       <string>Ключ</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="keyScrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="key">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>532</width>
            <height>326</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="evaluationMapTab">
      <attribute name="title">
       <string>Оценочная карта</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="evaluationMapScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="evaluationMap">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>523</width>
            <height>317</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
